zerver/openapi/zulip.yaml

```diff
@@ -8416,6 +8416,63 @@ paths:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
 
+  /users/{user_id}/status:
+    get:
+      operationId: get-user-status
+      summary: Get a user's status
+      tags: ["users"]
+      description: |
+        Get the [status](/help/status-and-availability) of a user.
```

Comment:
I feel like we should note this is something that's set by the user to somewhat highlight how this is different than user presence.

```
Get the [status](/help/status-and-availability) currently set by a user in the organization.
```